nano/node/election.cpp

```diff
@@ -790,3 +781,19 @@ void nano::election_extended_status::operator() (nano::object_stream & obs) cons
 		obs.write ("amount", amount);
 	});
 }
+
+/*
```

Comment:
Looks like a spurious comment.

Reply:
I added it to separate election class implementation from boilerplate enum conversions. I couldn't think of a good name for this section.

Reply:
> I added it to separate election class implementation from boilerplate enum conversions. I couldn't think of a good name for this section.

"Functions for converting election enums to strings and stat details"